docs/compute/persistent_tpm_and_uefi_state.md

```diff
@@ -22,9 +24,9 @@ spec:
       - VMPersistentState
 ```
 
-### Limitations
-
-- As mentioned above, the backend storage PVC can only be created using a storage class that supports RWX FS. There is ongoing work to support block storage in future versions of KubeVirt.
+### Notes:
+- If no storage class is specified, the default storage class will be used
+- If the storage class has a storage profile that indicates it only supports read-write-once (RWO) then a RWO PVC will be created and the VMI will be marked as non-migratable.
```

Comment:
Same as above

Reply:
I have pushed an update to address this.